packages/camera/camera_windows/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.2.6
 
 * Updates minimum supported SDK version to Flutter 3.19/Dart 3.3.
+* Fix black bars on camera preview [#122966](https://github.com/flutter/flutter/issues/122966)
```

Comment:
"Fixes". Also, this is missing a period. Please see the CHANGELOG style guide linked from the PR checklist.

Reply:
The changelog should now be following the style guide as of 4ffbedd